scripts/buildlsp.sh

```diff
@@ -1,18 +1,30 @@
 #!/bin/sh
+#
+# Builds the copilot lsp, then copies the generated dist/ folder to
+# marimo/_lsp/copilot
+
+# This script should fail if any intermediate step fails
+set -e
 
 cd lsp
 if pnpm build; then
   echo "Removing old lsp files..."
   rm -rf ../marimo/_lsp/
   echo "Copying new lsp files..."
-  mkdir ../marimo/_lsp
+  mkdir -p ../marimo/_lsp
   cp dist/index.cjs ../marimo/_lsp/
   # There seems to be a discrepancy between CI and mac for pnpm builds
   if [ -d dist/copilot ]; then
+    echo "Copying dist/copilot to _lsp/copilot"
     cp -R dist/copilot ../marimo/_lsp/copilot
+  elif [ -d dist/dist ]; then
+    echo "Copying contents of dist/dist/. to _lsp/copilot"
+    cp -R dist/dist/. ../marimo/_lsp/copilot/
   else
-    mkdir ../marimo/_lsp
-    cp -R dist ../marimo/_lsp/copilot
+    echo "Copying the contents of dist/. to _lsp/copilot"
+    # Do NOT place dist as a subdirectory in copilot/, just copy its contents;
+    # the period after dist/ is important here.
+    cp -R dist/. ../marimo/_lsp/copilot/
```

Comment:
Hmm, this almost confirms `rm -rf ../marimo/_lsp/` didn't clean up everything. This is a good catch all, but the folder should not already exist due to the` rm`